registry/txt.go

```diff
@@ -211,14 +223,15 @@ type nameMapper interface {
 }
 
 type affixNameMapper struct {
-	prefix string
-	suffix string
+	prefix              string
+	suffix              string
+	wildcardReplacement string
 }
 
 var _ nameMapper = affixNameMapper{}
 
-func newaffixNameMapper(prefix string, suffix string) affixNameMapper {
-	return affixNameMapper{prefix: strings.ToLower(prefix), suffix: strings.ToLower(suffix)}
+func newaffixNameMapper(prefix string, suffix string, wildcardReplacement string) affixNameMapper {
+	return affixNameMapper{prefix: strings.ToLower(prefix), suffix: strings.ToLower(suffix), wildcardReplacement: wildcardReplacement}
```

Comment:
I'm curious ... why not also call `strings.ToLower()` for the `wildcardReplacement`?

Reply:
@dansimone please let me know what you think when you have some time. Thanks!

Reply:
@seanmalloy sorry for the lag.  Yes, I agree.  Done in the latest commit.